content/20.pharmaceuticals.md

```diff
@@ -243,7 +244,23 @@ Patients who received HCQ also had a longer duration of hospitalization than pat
+One retrospective study reported that ivermectin reduced all-cause mortality [@doi:0.1016/j.chest.2020.10.009] while another reported no difference in clinical outcomes or viral clearance [@doi:10.1371/journal.pone.0242184].
+Small RCTs enrolling less than 50 patients per arm have also reported a wide array of positive [@doi:10.4103/1995-7645.318304; @doi:10.1093/qjmed/hcab035; @doi:10.1016/j.toxrep.2021.03.003; @doi:10.1016/j.ijid.2020.11.191; @doi:10.1016/j.clinthera.2021.04.007] and negative results [@doi:10.1016/j.eclinm.2020.100720; @doi:10.21203/rs.3.rs-191648].
+A slightly larger RCT enrolling 115 patients in two arms reported inconclusive results [@doi:10.18433/jpps32105].
+Hope for the potential of ivermectin peaked with the release of a preprint reporting results of a multicenter, double-blind RCT where a four-day course of ivermectin was associated with clinical improvement and earlier viral clearance in 400 symptomatic patients and 200 close contacts [@doi:https://doi.org/10.21203/rs.3.rs-100956/v3]; however, concerns were raised about both the integrity of the data and the paper itself [@url:https://grftr.news/why-was-a-major-study-on-ivermectin-for-covid-19-just-retracted; @url:https://steamtraen.blogspot.com/2021/07/Some-problems-with-the-data-from-a-Covid-study.html], and this study was removed by the preprint server Research Square [@doi:10.21203/rs.3.rs-100956/v4].
+A similarly sized RCT suggested no effect on the duration of symptoms among 400 patients split evenly across the intervention and control arms [@doi:10.1001/jama.2021.3071], and although meta-analyses have reported both null [@doi:10.1101/2021.05.21.2125759; @doi:10.1101/2021.01.26.21250420] and beneficial [@doi:10.1016/j.dsx.2021.102186; @doi:10.1097/MJT.0000000000001402; @doi:10.1002/rmv.2265; @doi:10.31219/osf.io/k37ft; @doi:10.1097/MJT.0000000000001377; doi:10.1007/s43440-021-00245-z; @doi:10.1101/2021.04.30.21256415] effects of ivermectin on COVID-19 outcomes, the certainty is likely to be low [@doi:10.1016/j.dsx.2021.102186] and these findings are potentially biased by a small number of low-quality studies, including the preprint that has been taken down [@url:https://gidmk.medium.com/does-ivermectin-work-for-covid-19-1166126c364a].
+Thus, much like HCQ/CQ, enthusiasm for research that either has not or should not have passed peer review has led to large numbers of patients worldwide receiving treatments that might not have any effect or could even be harmful.  
+Additionally, comments on the now-removed preprint include inquiries into how best to self-administer veterinary ivermectin as a prophylactic [@url:https://www.researchsquare.com/article/rs-100956/v4], and the FDA has posted information explaining why veterinary ivermectin should not be taken by humans concerned about COVID-19 [@url:https://www.fda.gov/animal-veterinary/product-safety-information/faq-covid-19-and-ivermectin-intended-animals].
```

Comment:
Was this the retracted version that was cited by DOI before?

Reply:
It is the same DOI, but the comments are on the landing page rather than in the document itself, so I wasn't sure whether citing the DOI made sense or whether it should be a URL!

Reply:
Can we use the same citation for both of the retracted v4 versions?  The DOI version has better meta-data.  When I visit https://doi.org/10.21203/rs.3.rs-100956/v4, I see the comments at the bottom, so I think `doi:10.21203/rs.3.rs-100956/v4` will work here as well.

`$ manubot cite doi:10.21203/rs.3.rs-100956/v4 --format markdown`
1. **Efficacy and Safety of Ivermectin for Treatment and prophylaxis of COVID-19 Pandemic**
Research Square Platform LLC *Research Square Platform LLC* (2020-10-30) <https://doi.org/gmhmfm>
DOI: [10.21203/rs.3.rs-100956/v4](https://doi.org/10.21203/rs.3.rs-100956/v4)
(manubot)

`$ manubot cite url:https://www.researchsquare.com/article/rs-100956/v4 --format markdown`
1. **Efficacy and Safety of Ivermectin for Treatment and prophylaxis of COVID-19 Pandemic** (2021-07-14) <https://www.researchsquare.com>

```suggestion
Additionally, comments on the now-removed preprint include inquiries into how best to self-administer veterinary ivermectin as a prophylactic [@doi:10.21203/rs.3.rs-100956/v4], and the FDA has posted information explaining why veterinary ivermectin should not be taken by humans concerned about COVID-19 [@url:https://www.fda.gov/animal-veterinary/product-safety-information/faq-covid-19-and-ivermectin-intended-animals].
```